SitePackage.lua

```diff
@@ -132,6 +132,26 @@ local function msg_hook(mode, output)
 
     dbg.print{"Mode is ", mode, "\n"}
 
+    if output[1] and output[1]:find("Your site prevents the automatic swapping of modules with same name") then
+        -- find the module name causing the issue (almost always toolchain module)
+        local sname  = output[1]:match("$ module swap ([^ ]+)%s+[^ \n]+")
+        local mStack = ModuleStack:moduleStack()
+
+        local errmsg = {"A different version of the '"..sname.."' module is already loaded (see output of 'ml')."}
+        if not mStack:empty() then
+            errmsg[#errmsg+1] = " You should load another '"..mStack:sn().."' module for that is compatible with the "
+            errmsg[#errmsg+1] = "currently loaded version of '"..sname.."'. Use 'ml spider "..mStack:sn().."' to get "
+            errmsg[#errmsg+1] = "an overview of the available versions."
```

Comment:
@wpoely86 start every sentence on a new line, to try and make sure people will actually *read*